docs/integrations/community-ecosystem-apps.md

```diff
@@ -23,19 +23,75 @@ The following links provide collection, installation, and dashboard information
+
+[Symantec Web Security Service (WSS)](https://github.com/SumoLogic/sumologic-content/tree/master/Symantec/WSS)
+
+[Sumo Logic Traces](https://github.com/SumoLogic/sumologic-content/tree/master/Sumo_Logic_Native/Traces)
+
+[Tenable.io](https://github.com/SumoLogic/sumologic-content/tree/master/Tenable/Tenable_IO)
+
+[Qualys VMDR & Inventory](https://github.com/SumoLogic/sumologic-content/tree/master/Qualys)
+
+
+## Contributing Guidelines
+
+Content can include vendor-based/custom applications, dashboards, search queries, field extraction rules (FERs), tooling, etc. Please complete each of the required fields in the [Sumo Logic Community Content Submission](https://forms.gle/KQBLBuMuUw85xtRi9) form, and follow the guidelines below:
+- Verify that all sensitive data has been removed or redacted BEFORE submitting content. This includes PII, Credentials, Network data, etc. If you provided screenshots, either blur or mark over this sensitive data.
+- All application, dashboard, and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive naming convention for files (Company-Product-Function. Ex: AWS-Kinesis-Errors.json).
+- When possible, include a screenshot of your dashboards in .png or .jpg format. Name your screenshots the same as their respective JSON content.
+- If you have notes on how to collect data for specific content, please provide those steps as well.
+- If you have a GitHub Repository for your content already, please feel free to provide the link to this repository, and make sure it has Public access.
+
+Once submitted, Sumo Logic will review your content for publishing to our Community Ecosystem repository.
+
+### Alternative Submission
+
+Follow the "fork-and-pull" Git workflow:
+- **Fork** the repo on GitHub
+- **Clone** the project to your own machine
+- **Create** any new folders/subfolders necessary for your content
+	- Add your content to these folders
+	- Verify that all sensitive data has been removed  or redacted BEFORE submitting a PR. This includes PII, Credentials, and Network data. Screenshots included (either blur or mark over).
+	- All application, dashboard and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive naming convention (Company-Product-Function. Ex: AWS-Kinesis-Errors.json)
+	- When possible, include a screenshot of your dashboards in .png or .jpg format. Name your screenshots the same as their respective JSON content. If there is more than one, please place these images in a Screenshots folder.
+	- Create README.md file within each subfolder to track descriptions of the app, guidance on setting up the data collection, authors, versions, dates, and links to 3rd party docs
+- **Commit** the changes on your local machine to your own fork/branch
+- **Push** your work back up to your fork
+- **Submit** a pull request so that we can review your changes and publish
```

Comment:
```suggestion
Follow the "fork-and-pull" Git workflow:
- **Fork** the repo on GitHub.
- **Clone** the project to your own machine.
- **Create** any new folders/subfolders necessary for your content.
   - Add your content to these folders.
   - Verify that all sensitive data has been removed or redacted BEFORE submitting a PR. This includes PII, Credentials, and Network data. Screenshots included (either blur or mark over).
   - All application, dashboard and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive name, ideally the naming convention <company>-<product>-<function> (example: AWS-Kinesis-Errors.json).
   - When possible, include a screenshot of your dashboards in .png or .jpg format. Name your screenshots the same as their respective JSON content. If there is more than one, please place these images in a Screenshots folder.
   - Create README.md file within each subfolder to track descriptions of the app, guidance on setting up the data collection, authors, versions, dates, and links to third party docs.
- **Commit** the changes on your local machine to your own fork/branch.
- **Push** your work back up to your fork.
- **Submit** a pull request so that we can review your changes and publish.
```